### PR TITLE
Adds CI for WPF Unit Tests

### PR DESCRIPTION
**Pull request by klaki892**
Adds Github Action to preform unit tests that dont need a online registry.

When merged, Github will preform unit test checking on any new PR that gets generated

### CODE REVIEW

**Comment by klaki892** [comment]
Surprisingly i guess it doesnt need to be merged to be activated. Obviously merging will activate the test for all PR's and not just this one.

It appears that by default Github will notify you by email of a failed action (and there's no real configuration to ignore one specific action.) [you can turn that email notification off here](https://github.com/settings/notifications)

**Comment by szczura** [comment]
So should we merge this now?

**Comment by klaki892** [comment]
> So should we merge this now?

Once People approve it, yes.

File: WPF/SeeShells/SeeShellsTests/ShellParser/OnlineRegistryReaderTests.cs [hunk 1]
```diff
@@ -12,6 +12,7 @@ public class OnlineRegistryReaderTests
         /// Tests that the Online Registry Reader can read from a running system's registry.
         /// </summary>
         [TestMethod()]
+        [TestCategory("OnlineTest")]
```
Comment (AlekStoyanov) [inline WPF/SeeShells/SeeShellsTests/ShellParser/OnlineRegistryReaderTests.cs]: Is this `[TestCategory("OnlineTest")]` tag added to make github ignore said test?

Comment (klaki892) [reply WPF/SeeShells/SeeShellsTests/ShellParser/OnlineRegistryReaderTests.cs]: Yes, its meant to ignore any test that relies on the registry having something
as ive discovered Github CI's virtual machines dont have anything in there, which causes these tests to fail

**Pull request merged**